src/components/Apps/AccountListTable/AccountList.vue

```diff
@@ -311,7 +311,7 @@ export default {
             title: this.$t('TemplateAdd'),
             has: !(this.platform || this.asset),
             can: () => {
-              return vm.$hasPerm('accounts.add_account') && !this.$store.getters.currentOrgIsRoot
+              return vm.$hasPerm('accounts.add_account') && !vm.$store.getters.currentOrgIsRoot
             },
             callback: async() => {
               await this.getAssetDetail()
```

Comment:
I cannot review this code since it does not have actual lines of text. Please provide the relevant sections or the entire file so I could analyze it.